src/app/components/repertoire/repertoire.component.ts

```diff
@@ -52,12 +78,18 @@ export class RepertoireComponent implements OnInit {
 
     getSeasonPerformancesSub$
       .subscribe((performances) => {
-        this.perfomances = performances;
-        this.collectionSize = performances.length;
-        this.changeDetector.markForCheck();
-        this.loaderService.stop('repertoire');
-      },
-      error1 => this.loaderService.stop('repertoire')
-    );
+          this.performances = performances;
+          this.collectionSize = performances.length;
+          this.changeDetector.markForCheck();
+          this.loaderService.stop('repertoire');
+        },
+        error1 => this.loaderService.stop('repertoire')
+      );
+  }
+
+  ngOnDestroy() {
+    if (this.scrollToBottomDuePagination) {
```

Comment:
scrollToBottomDuePagination - use better names for props...

Reply:
Fixed